app/assets/javascripts/analytics/static-tracker.js

```diff
@@ -78,6 +79,26 @@
       tracker.setDimension(customVar[1], customVar[3], customVar[2], customVar[4]);
     }
 
+    function setDimensionsFromMetaTags() {
```

Comment:
Adding it to the public api would let us add proper unit tests to just this method. Which if it is private we can't do. Also while there is no need for this to be added to the public api there is also no harm in it being public. I would lean on the side of making JavaScript methods public unless there is a good reason not to.